software/SolarAISVN/SolarAISVN.ino

```diff
@@ -161,14 +164,18 @@ void measureVoltages() {
   //    32,      33,       34,       35,   14,   26,   27,     12,   13
   // solar, battery, currentA, currentB, load, wind, dump, solar2, LiPo  
   voltage[0] = int((4096 - voltage[0]) * 7.52 - 1000);  // pin32 solar    voltage divider 10k : 1.2 k Ohm 1:1
+  if(voltage[0] < 0) voltage[0] = 0;
```

Comment:
Negative voltage would be adjusted to zero.

Reply:
Good update!